releng/org.eclipse.xtext.contributor/Xtext.setup

```diff
@@ -47,11 +47,11 @@
   <setupTask
       xsi:type="setup:VariableTask"
       name="xtext.nightly.composite"
-      value="http://services.typefox.io/open-source/jenkins/job/xtext-eclipse/job/master/lastStableBuild/artifact/build/p2-repository/"/>
+      value="http://services.typefox.io/open-source/jenkins/job/xtext/job/master/lastStableBuild/artifact/build/p2-repository/"/>
   <setupTask
       xsi:type="setup:VariableTask"
       name="xtend.nightly.composite"
-      value="http://services.typefox.io/open-source/jenkins/job/xtext-xtend/job/master/lastSuccessfulBuild/artifact/build/p2-repository/"/>
+      value="http://services.typefox.io/open-source/jenkins/job/xtext-xtend/job/master/lastStableBuild/artifact/build/p2-repository/"/>
```

Comment:
I think you should only use the single aggregate repo, http://services.typefox.io/open-source/jenkins/job/xtext/job/master/lastSuccessfulBuild/artifact/build/p2-repository/ and not two repos (one for Xtext and one for Xtend).